docs/reference/commandline/cli.md

```diff
@@ -117,18 +117,18 @@ if no `--format` flag is provided.
 
 | Property               | Description                                                                                                                                                         |
 | :--------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `configFormat`         | Custom default format for `docker config ls` output. See [`docker config ls`](config_ls.md#format) for a list of supported formatting directives.                   |
-| `imagesFormat`         | Custom default format for `docker images` / `docker image ls` output. See [`docker images`](image_ls.md#format) for a list of supported formatting directives.      |
-| `networksFormat`       | Custom default format for `docker network ls` output. See [`docker network ls`](network_ls.md#format) for a list of supported formatting directives.                |
-| `nodesFormat`          | Custom default format for `docker node ls` output. See [`docker node ls`](node_ls.md#format) for a list of supported formatting directives.                         |
-| `pluginsFormat`        | Custom default format for `docker plugin ls` output. See [`docker plugin ls`](plugin_ls.md#format) for a list of supported formatting directives.                   |
-| `psFormat`             | Custom default format for `docker ps` / `docker container ps` output. See [`docker ps`](container_ls.md#format) for a list of supported formatting directives.      |
-| `secretFormat`         | Custom default format for `docker secret ls` output. See [`docker secret ls`](secret_ls.md#format) for a list of supported formatting directives.                   |
-| `serviceInspectFormat` | Custom default format for `docker service inspect` output. See [`docker service inspect`](service_inspect.md#format) for a list of supported formatting directives. |
-| `servicesFormat`       | Custom default format for `docker service ls` output. See [`docker service ls`](service_ls.md#format) for a list of supported formatting directives.                |
-| `statsFormat`          | Custom default format for `docker stats` output. See [`docker stats`](container_stats.md#format) for a list of supported formatting directives.                     |
-| `tasksFormat`          | Custom default format for `docker stack ps` output. See [`docker stack ps`](stack_ps.md#format) for a list of supported formatting directives.                      |
-| `volumesFormat`        | Custom default format for `docker volume ls` output. See [`docker volume ls`](volume_ls.md#format) for a list of supported formatting directives.                   |
+| `configFormat`         | Custom default format for `docker config ls` output. See [`docker config ls`](https://docs.docker.com/reference/cli/docker/config/ls/#format) for a list of supported formatting directives.                   |
```

Comment:
🤔 honestly, not sure if this is the right thing to do. We were using `docs.docker.com` URLs for content maintained _outside of this repository_, but for content maintained here ("locally") using docs.docker.com links will be problematic.


- it's hard to verify (e.g. IDEs won't be able to validate anchors)
- we end up with a chicken-and-egg situation; we must add "broken links" for new content, as well as changes that we make (docs.docker.com won't be updated until we release and publish content).


I would _NOT_ be against moving away from a flat-structure though. We have a lot of commands in many groups, so organising those in directories `image/pull.md` and so on) would work for me if that allows use to continue using links to the markdown files.

We may have to look at the `cli-docs-tool` to facilitate this though!

Reply:
The problem with the problem with creating a nested structure with subdirectories would be that we'd have lots of `docs/reference/image/_index.md` files as well laying around. And it makes it much more involved to generate to/from these files. I think it'd be a lot more straightforward just having absolute links.

Reply:
I've pushed a patch to cli-docs-tool that handles the URL path conversion when generating yaml: https://github.com/docker/cli-docs-tool/pull/49

So I reverted back to relative paths where possible. The `dockerd.md` and a couple of other files that aren't part of the cli reference are exempted. That's because they'll be served under a different url path entirely so the preprocessing step in `cli-docs-tool` doesn't work in those cases.